cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -154,6 +154,20 @@ Known Issues
     
   All versions of CDAP running Twill version 0.4.0 with this configuration can exhibit this
   problem (`TWILL-110 <https://issues.apache.org/jira/browse/TWILL-110>`__).
+- Typically Datasets are bundled as part of Applications. When an Application is upgraded and redeployed,
+  any changes in Datasets will not be redeployed. This is because Datasets can be shared across applications,
+  and an incompatible schema change can break other applications that are using the Dataset.
+  A workaround (`CDAP-1253 <https://issues.cask.co/browse/CDAP-1253>`__) is to allow unchecked Dataset upgrades.
+  Upgrades cause the Dataset metadata i.e. it's specification, including properties, to be updated. The Dataset
+  runtime code is also updated. In addition, to prevent data loss the existing data and the
+  underlying HBase tables remain as is. You can do this by setting the configuration
```

Comment:
do this? prevent data loss? 
I think it should be a new paragraph, and then: You can allow unchecked Dataset upgrades by...